.tool-versions.sample

```diff
@@ -1,3 +1,3 @@
 ruby 2.6.10
 nodejs 12.22.12
-python 3.10.7 2.7.18 # Meant for M1 macs
+python 2.7.18
```

Comment:
what requires python 2?

Reply:
Something failed at some point with python 3, now I don't know what it was anymore. There are too many pieces in this crazy puzzle.